HHS/Head-Start-TTADP
# [TTAHUB-3309] Only one connection per objective to ARO

## Description of change

- Fix hole that created duplicate ARO entries when a file was uploaded
- migration to merge duplicate aro records for report from support case: ActivityReportId: 46116 

## How to test

1. On a report with and objective
2. Save
3. Add a file
4. Save
5. Add a file
6. Check the DB to confirm only a single ARO is on the report

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-0


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
